mypyc/lib-rt/tuple_ops.c

```diff
@@ -55,7 +55,7 @@ bool CPySequenceTuple_SetItemUnsafe(PyObject *tuple, CPyTagged index, PyObject *
         PyTuple_SET_ITEM(tuple, n, value);
         return true;
     } else {
-        PyErr_SetString(PyExc_OverflowError, "Python int too large to convert to C ssize_t");
+        PyErr_SetString(PyExc_OverflowError, "set_item index too large for tuple");
```

Comment:
Hmm since the original error message was duplicated in multiple places, a better approach would be to use a `#define` for the error message. 

That being said, it looks like the original message was copied verbatim from CPython, so I now think that it would better to preserve the old message (here and elsewhere). Some existing test cases could test the exception message, and even if the new message is more informative, we could still break some existing tests, which we typically want to avoid.

If you add a `#define` and then add a comment saying that the message is copied from CPython and shouldn't be changed, things will be clearer in the future.